explainers/hidden-content-explainer.md

```diff
@@ -1,18 +1,15 @@
-# Finding information in hidden sections.
+# `content-visibility: hidden-matchable` and the `beforematch` event
 
-## content-visibility: hidden-matchable and the `beforematch` event.
+## Summary (TL;DR)
 
-### Summary (TL;DR)
+This is an explainer for two coupled features:
 
-This is an explainer for two closely related features:
+1. `content-visibility: hidden-matchable` is a new value for the CSS property
+   `content-visibility`. It makes the content hidden from the user, similar to
+   `content-visibility: hidden`, but the content remains searchable via
+   user-agent algorithms such as find-in-page.
 
-1. Content visibility hidden matchable is an extension to the `content-visibility`
-    feature. It adds a new value to the possible set of properties:
-    * `content-visibility: hidden-matchable`: in this configuration, the content is
-        hidden from the user (similar to `content-visibility: hidden`), but the
-        content remains searchable via user-agent find-in-page algorithms.
-
-2. The `beforematch` event allows developers to display `hidden-matchable`
+2. The `beforematch` event allows developers to reveal `hidden-matchable`
     content to the user in response to searches which scroll the page to some
     target text. The event is fired on the nearest `content-visibility:
     hidden-matchable` ancestor at render timing for these cases:
```

Comment:
for the find-in-page, can you add a blurb that this includes window.find

Reply:
Just finished reading the rest of the changes: I think we should still say window.find here with a note saying that this is discussed more below

Reply:
done, how does it look?

Reply:
lgtm